SUMMARY.md

```diff
@@ -31,4 +31,6 @@
 * [Deploy the application](chapters/deploy-the-application/README.md)
     * [Deploy to Azure](chapters/deploy-the-application/deploy-to-azure.md)
     * [Deploy with Docker](chapters/deploy-the-application/deploy-with-docker.md)
+* [Alternative databases](chapters/alternative-databases/README.md)
```

Comment:
I'm still trying to decide where this should go. I think it makes more sense before Deploy, but then the Deploy chapter might have to worry about which database people are using.

Reply:
Awaiting feedback on proposal in comment below.